src/Application/Response/UserCreationResponse.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Application\Response;
+
+class UserCreationResponse implements \JsonSerializable
+{
+    private $userId;
+    private $userName;
+
+    public function __construct(int $userId, string $userName)
+    {
+        $this->userId = $userId;
+        $this->userName = $userName;
+    }
+
+    public function getUserId()
+    {
+        return $this->userId;
+    }
+
+    public function getUserName()
+    {
+        return $this->userName;
+    }
```

Comment:
This is not a critique, just a comment, lately there's a trend that is removing the word "get" from the getters, since is a little bit redundant.

We follow that trend in Diet's project, but is up to you if you want to do it too!

Reply:
then what is is trend to replace the get?